bin/install.sh

```diff
@@ -177,12 +177,23 @@ else
     ANSIBLE_VERSION=ansible==2.8.8
 fi
 
+RASPBIAN_VERSION=$(cat /etc/os-release | grep VERSION= | cut -d'=' -f2 | cut -d'"' -f2 | cut -d' ' -f1)
```

Comment:
Just use `lsb_release` instead.

Reply:
Running `lsb_release` just returns `No LSB modules are available.`

Reply:
What about if you for '-a' and play with various flags?

Reply:
Oh, wait. Running `lsb_release -a` displays the distro version.

Reply:
@vpetersson. I resorted to just using `lsb_release -rs`, and it works. Now it's much simpler and cleaner. Thanks!